policy/loader.go

```diff
@@ -6,58 +6,59 @@ import (
 
 	"github.com/open-policy-agent/conftest/downloader"
 
-	"github.com/open-policy-agent/opa/storage"
+	"github.com/open-policy-agent/opa/loader"
 )
 
 // Loader handles the retrieval of all rego policies and related data.
 type Loader struct {
 	PolicyPaths []string
 	DataPaths   []string
 	URLs        []string
-
-	test bool
+	Tracing     bool
```

Comment:
Is Tracing defined on the loader just to configure it as part of the engine? Would it not make more sense to have a configuration method on the engine? E.g. `Engine.SetTracing(true)`

Reply:
I didn't like this flag on the engine when I first did it :( Today I'm going to try the approach of: always include trace data, but only render it if the trace flag is on. Make it the responsibility of the output format.